content/applications/general/users/delete_account.rst

```diff
@@ -0,0 +1,35 @@
+===========================
+Odoo.com Account Management
+===========================
+
+Account management
+==================
+
+Overview
+--------
+
+To delete your Odoo account, access the delete option by clicking on the user icon. The delete
+option can be accessed by going to :menuselection:`My Account --> Edit Security Settings -->
+Delete Account`. It can also be accessed by going to https://www.odoo.com/my/home.
+
+Deletion Process
+----------------
+
+Click on the user icon and select :guilabel:`My Account`. Then click on :guilabel:`Edit Security
+Settings`.
+
+Next, under the :guilabel:`Delete Account` section, click on the :guilabel:`Delete Account` button.
+
+.. danger::
+   Deleting an Odoo account is irreversible. Take caution when performing this action as the Odoo.com
+   account will not be retrievable.
+
+Upon clicking the :guilabel:`Delete Account` button Odoo will populate a pop-up window verifying the
+change to delete the account.
```

Comment:
Sentence reads weird imo.

Wording suggestion to simplify it:
```suggestion
Upon clicking the :guilabel:`Delete Account` button, a pop-up window will appear, asking you to
confirm the account deletion.
```

Reply:
👍 All changed! Thanks.